sql/world/uai_WorldSpellDBC.sql

```diff
@@ -0,0 +1,8 @@
+-- use world;
+INSERT INTO `spell_dbc` VALUES (80865,0,0,256,268468224,0,0,0,0,0,0,0,0,0,1,0,0,101,0,0,0,0,21,1,0,-1,0,0,6,0,0,1,0,0,0,0,0,0,0,0,0,0,0,25,0,0,0,0,0,0,0,0,77,0,0,0,0,0,0,0,0,0,0,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1,1,1,0,1,'AutoBalance Immune Charm');
+INSERT INTO `spell_dbc` VALUES (80866,0,0,256,268468224,0,0,0,0,0,0,0,0,0,1,0,0,101,0,0,0,0,21,1,0,-1,0,0,6,0,0,1,0,0,0,0,0,0,0,0,0,0,0,25,0,0,0,0,0,0,0,0,77,0,0,0,0,0,0,0,0,0,0,0,5,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1,1,1,0,1,'AutoBalance Immune Fear');
+INSERT INTO `spell_dbc` VALUES (80867,0,0,256,268468224,0,0,0,0,0,0,0,0,0,1,0,0,101,0,0,0,0,21,1,0,-1,0,0,6,0,0,1,0,0,0,0,0,0,0,0,0,0,0,25,0,0,0,0,0,0,0,0,77,0,0,0,0,0,0,0,0,0,0,0,7,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1,1,1,0,1,'AutoBalance Immune Root');
+INSERT INTO `spell_dbc` VALUES (80868,0,0,256,268468224,0,0,0,0,0,0,0,0,0,1,0,0,101,0,0,0,0,21,1,0,-1,0,0,6,0,0,1,0,0,0,0,0,0,0,0,0,0,0,25,0,0,0,0,0,0,0,0,77,0,0,0,0,0,0,0,0,0,0,0,11,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1,1,1,0,1,'AutoBalance Immune Snare');
+INSERT INTO `spell_dbc` VALUES (80869,0,0,256,268468224,0,0,0,0,0,0,0,0,0,1,0,0,101,0,0,0,0,21,1,0,-1,0,0,6,0,0,1,0,0,0,0,0,0,0,0,0,0,0,25,0,0,0,0,0,0,0,0,77,0,0,0,0,0,0,0,0,0,0,0,12,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1,1,1,0,1,'AutoBalance Immune Stun');
+INSERT INTO `spell_dbc` VALUES (80870,0,0,256,268468224,0,0,0,0,0,0,0,0,0,1,0,0,101,0,0,0,0,21,1,0,-1,0,0,6,0,0,1,0,0,0,0,0,0,0,0,0,0,0,25,0,0,0,0,0,0,0,0,77,0,0,0,0,0,0,0,0,0,0,0,17,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1,1,1,0,1,'AutoBalance Immune Polymorph');
+INSERT INTO `spell_dbc` VALUES (80871,0,0,256,268468224,0,0,0,0,0,0,0,0,0,1,0,0,101,0,0,0,0,21,1,0,-1,0,0,6,0,0,1,0,0,0,0,0,0,0,0,0,0,0,25,0,0,0,0,0,0,0,0,77,0,0,0,0,0,0,0,0,0,0,0,27,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1,1,1,0,1,'AutoBalance Immune Daze');
```

Comment:
Put a `DELETE` before a `INSERT`.
http://www.azerothcore.org/wiki/Dealing-with-SQL-files

Reply:
With regards to core DB tables and data, do we really want modules deleting things? To protect core data integrity shouldn't we use:

INSERT IGNORE instead? (because in this case, I'm taking a block of spell IDs from the spell.dbc table). Thoughts?

Reply:
Maybe you're right, we can in this case because we are using spell_dbc, but usually I always write DELETE FROM also to prevent different scenarios on how I import (eventually) the sql code / dump.

Reply:
yeah, only INSERT in such case imo (already did it like that on some other modules when creating a new NPC)